packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -1400,6 +1417,11 @@ export class AccountHandler {
       res.authenticatorAssuranceLevel =
         authMethods.maximumAssuranceLevel(amrValues);
     }
+    if (scope.contains('profile:age_check')) {
+      res.atLeast18AtReg = account.atLeast18AtReg
+        ? account.atLeast18AtReg
+        : null;
```

Comment:
This ternary is unnecessary.  `account.atLeast118AtRge` is already the value we want.

Reply:
hmm maybe this is not true since I also see another ternary in the profile-server and I think I was seeing result of that not this.  Confusing.  😖